data/changelog/odoc/2021-10-06-odoc-2.0.0.md

```diff
@@ -110,23 +110,23 @@ We look forward to many new renderers being created for the varied use cases pre
+Like the OCaml compiler itself, running `odoc` on your code requires careful sequencing of the invocations to produce the correct result. Fortunately both `dune` and `odig` understand how to do this, so most users don't need to know the details. If you want more than these tools provide though, we've written a simple [reference driver](https://ocaml.github.io/odoc/driver.html - [404 Not Found]), documenting exactly what's necessary to use `odoc` to produce rich documentation. A more complete (and more complex) example is the tool [voodoo](https://github.com/ocaml-doc/voodoo), which is being used to create the docs for [v3.ocaml.org](https://v3.ocaml.org/packages).
 
 ## [v3.ocaml.org](https://v3.ocaml.org)
 
 As previously posted, the new version of the OCaml website has been under development for some time now, and an important new feature is the integration of package listings, including documentation for every version of every package. More has been written about this elsewhere, but it's important to note that the new OCaml.org website required a preview version of `odoc 2.0` to work. We've made a few bug fixes since then, so we will update the pipeline to use the released version very soon. For more info on the pipeline to build the docs, see [our recent talk](https://watch.ocaml.org/videos/watch/9bb452d6-1829-4dac-a6a2-46b31050c931) at this year's OCaml Workshop.
 
 ## New Website
 
-The website for `odoc` has been improved with guides for [documentation authors](https://ocaml.github.io/odoc/odoc_for_authors.html), [integrators](https://ocaml.github.io/odoc/driver.html), and [contributors](https://ocaml.github.io/odoc/contributing.html). This site is intended to grow over time with more content to help people write docs for their packages.
+The website for `odoc` has been improved with guides for [documentation authors](https://ocaml.github.io/odoc/odoc_for_authors.html - [404 Not Found]), [integrators](https://ocaml.github.io/odoc/driver.html - [404 Not Found]), and [contributors](https://ocaml.github.io/odoc/contributing.html - [404 Not Found]). This site is intended to grow over time with more content to help people write docs for their packages.
 
 ## OCamldoc?
 
-This release, particularly because of the new output renderers, puts `odoc` in a place where it supercedes OCamldoc in most respects. There are a few features we're missing (see [the comparison](https://ocaml.github.io/odoc/ocamldoc_differences.html) in the docs), including
+This release, particularly because of the new output renderers, puts `odoc` in a place where it supercedes OCamldoc in most respects. There are a few features we're missing (see [the comparison](https://ocaml.github.io/odoc/ocamldoc_differences.html - [404 Not Found]) in the docs), including
```

Comment:
```suggestion
This release, particularly because of the new output renderers, puts `odoc` in a place where it supercedes OCamldoc in most respects. There are a few features we're missing (see [the comparison](https://web.archive.org/web/20211208101345/https://ocaml.github.io/odoc/ocamldoc_differences.html) in the docs), including
```